packages/x-date-pickers/src/internals/hooks/useField/useFieldV6TextField.ts

```diff
@@ -88,7 +88,7 @@ export const useFieldV6TextField: UseFieldTextField<false> = (params) => {
       inputRef: inputRefProp,
       placeholder: inPlaceholder,
     },
-    internalProps: { readOnly = false },
+    internalProps: { readOnly = false, disabled },
```

Comment:
WDYT about aligning the approach and initializing the boolean to a default value as with `readOnly`?
```suggestion
    internalProps: { readOnly = false, disabled = false },
```